app/dashboard/notifications.py

```diff
@@ -717,7 +717,7 @@ def maybe_market_tip_to_github(tip):
         addon_msg = f"\n\n * ${amount_usdt_open_work()} in Funded OSS Work Available at: " \
                     f"https://gitcoin.co/explorer\n * Incentivize contributions to your repo: " \
                     f"<a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>" \
-                    f"Fund a PR</a>\n * No Email? Get help on the <a href='https://chat.gitcoin.co/'>Gitcoin Chat</a>"
+                    f"Fund a PR</a>\n * No Email? Get help on the <a href='https://discord.com/invite/83BK5z55yB'>Gitcoin Discord</a>"
```

Comment:
I think this should be Gitcoin's Discord. What do you think?

Reply:
oh yea.. that's right!